karma.conf.js

```diff
@@ -150,17 +25,15 @@ module.exports = function (config) {
 
     // list of files / patterns to load in the browser
     // all dependancies should be traced through here
-    files: [ 'test/unit.js' ],
+    files: files,
 
     // list of files to exclude
     exclude: [],
 
     // preprocess matching files before serving them to the browser
```

Comment:
either move this comment with the block or remove it (I think it can be removed)